### PR TITLE
One click deploy for platform.sh

### PR DESCRIPTION
**Pull request by OriPekelman**
CPO @ Platform.sh here, as I find Posthog to be an awesome idea, and am thinking about using it ... well, if I already deployed it to platform.sh I thought, I could go the extra mile and create a one-click deployment. So here you go.

## Changes

- Added platform.sh configurations files
- Added platform.sh to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,10 +49,16 @@ There are a couple of session-based open source libraries that are nice alternat
 
 You can [sign up for a free account](https://app.posthog.com/signup) on our hosted platform.
 
-## Deploy to Heroku
+## Deploy to PaaS
 
+### Heroku Install
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/posthog/posthog)
 
+### Platform.sh Install
```
Comment (timgl) [inline README.md:57]: We list all of our alternative ways of deploying [here](https://posthog.com/docs/deployment), which you can [edit here](https://github.com/PostHog/posthog.com/blob/master/contents/docs/deployment.md).

**Review by timgl — COMMENTED** [review]
Thanks so much for raising this PR! 🎉 One small comment.

File: README.md (hunk 1 above)
Comment (OriPekelman) [reply README.md]: Ok, removed the reference from the main README file and put more detailed instructions in a .platform/README.md file, and contributed to the other docs.

**Review by fuziontech — APPROVED** [review]
lgtm 👍

**Pull request merged**